TestAssets/TestProjects/PortableApp/project.json

```diff
@@ -9,7 +9,7 @@
       "dependencies": {
         "Microsoft.NETCore.App": {
           "type": "platform",
-          "version": "1.0.0"
+          "version": "1.0.0-servicing-004500"
```

Comment:
@ericstj, @brthor is right that once this commit goes to the feed, NuGet will always resolve `1.0.0-servicing-004500` against this commit, effectively rendering the tests useless.